Source/Source/CoppaStatus.cs

```diff
@@ -0,0 +1,50 @@
+
+namespace ScaleMonk.Ads
+{
+
+  /// <summary>
+  /// Represents the different COPPA status handled in the Scalemonk SDK.
+  /// </summary>
+  public enum CoppaStatus {
+    
+    /// <summary>
+    /// Unknown status.
+    /// </summary>
+    UNKNOWN,
+
+    /// <summary>
+    /// treat as child status.
+    /// </summary>
+    CHILD_TREATMENT_FALSE,
+
+    /// <summary>
+    /// Dont treat as child status.
+    /// </summary>
+    CHILD_TREATMENT_TRUE,
+
+    /// <summary>
+    /// Does not apply status.
+    /// </summary>
+    NOT_APPLICABLE
+  }
+
+  public static class CoppaStatusExtensions
```

Comment:
@christiangelone here I think you should make the conversion, since enums in C# are automatically ints.

Reply:
@marinoluck WDYT ?

Reply:
I prefer to keep it explicit. If someone changes the order of the enum will break everything and won't notice it.